wolfcrypt/src/siphash.c

```diff
@@ -744,6 +744,12 @@ int wc_SipHash(const unsigned char* key, const unsigned char* in, word32 inSz,
         "cmp    %w[outSz], #8\n\t"
         "b.eq   L_siphash_8_end\n\t"
 
+    : [in] "+r" (in), [inSz] "+r" (inSz)
```

Comment:
Sean, this broke the ARMv8 with `--enable-debug`. Can you review this section for @lealem47 ?

Reply:
Looking into it.
I will put up.a PR that doesn't break --enable-debug on M1.

Reply:
Fix: #6246